test/Mono.Linker.Tests.Cases/Inheritance.VirtualMethods/VirtualMethodGetsPreservedIfBaseMethodGetsInvoked.cs

```diff
@@ -23,7 +23,7 @@ public virtual void Foo ()
 		[KeptBaseType (typeof (B))]
 		class A : B
 		{
-			[Kept]
+			[KeptBy ("Mono.Linker.Tests.Cases.Inheritance.VirtualMethods.VirtualMethodGetsPreservedIfBaseMethodGetsInvoked/A", "OverrideOnInstantiatedType")]
```

Comment:
Could this be `typeof(A)` instead of the fully qualified type name?  It's nice to keep string names, especially fully qualified names, out of these attributes whenever possible.